.github/workflows/build_python.yml

```diff
@@ -0,0 +1,30 @@
+name: build_python
+# build for python, skip scala tests and R build
+on:
+  push:
+    branches:
+      - "python/*"
+  pull_request:
+    branches:
+      - "python/*"
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+    env:
+      GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
+    strategy:
+      matrix:
+        python: [ 3.8.10 ]
+        spark: [ 3.2.1 ]
+        R: [ 4.1.2 ]
+    steps:
+      - name: checkout code
+        uses: actions/checkout@v2
+      - name: build scala with skipping tests
```

Comment:
delete

Reply:
I kept scala in case we want a custom jar in the future for custom matcher functions. but fair point, we can add it back in when ready